tests/attestation_cert_apis/enc/enc.cpp

```diff
@@ -236,35 +242,50 @@ oe_result_t get_tls_cert_signed_with_key(
         goto done;
     }
 
-    // validate cert with oe_verify_attestation_certificate_with_evidence()
+    // validate cert with oe_verify_attestation_certificate_with_evidence_v2()
     // to ensure that the added report verifier part of the function works well
-    result = oe_verify_attestation_certificate_with_evidence(
-        output_cert, output_cert_size, enclave_claims_verifier, nullptr);
+    result = oe_verify_attestation_certificate_with_evidence_v2(
```

Comment:
Should we still proceed even `result != OE_OK` here? As `result` will be overwritten later.
I noticed previous code did proceed even verification failed here but not sure any specific reason for that. Otherwise we should `goto done` like Line 242.